tests/message/test_tasks.py

```diff
@@ -115,3 +121,13 @@ def test_concent_enabled_default_true(self):
     def test_concent_enabled_false(self):
         ttc = message.tasks.TaskToCompute(concent_enabled=False)
         self.assertFalse(ttc.concent_enabled)
+
+    def test_ethereum_address(self):
+        msg = factories.TaskToComputeFactory()
+        serialized = shortcuts.dump(msg, None, None)
+        msg_l = shortcuts.load(serialized, None, None)
+        for addr_slot in (
+                'requestor_ethereum_address',
+                'provider_ethereum_address'):
+            address = getattr(msg_l, addr_slot)
+            self.assertEqual(len(address), 2 + (20*2))
```

Comment:
Wouldn't it be worth to compare `requestor_ethereum_public_key` and `provider_ethereum_public_key` before and after serialization ?

Reply:
Actually it's already done in `SerializationMixin`.